WORKSPACE

```diff
@@ -139,6 +139,17 @@ load("@com_github_bazelbuild_buildtools//buildifier:deps.bzl", "buildifier_depen
 
 buildifier_dependencies()
 
+http_archive(
+    name = "com_github_prysmaticlabs_go_ssz",
+    sha256 = "a0e2ed9be1fc4fbc85626a7bb0d43a63db2c16d3a735565df5c6bde24678b304",
+    strip_prefix = "go-ssz-993cc801b0d7a95cce0ca8f9f24f90ae87b3d9ba",
+    url = "https://github.com/prysmaticlabs/go-ssz/archive/993cc801b0d7a95cce0ca8f9f24f90ae87b3d9ba.tar.gz",
```

Comment:
This is already defined on line 176, please update the commit on line 176 and remove these new lines

Reply:
done